samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java

```diff
@@ -562,4 +569,33 @@ public void testCloseShouldTearDownZkConnectionOnInterruptedException() throws E
 
     Assert.assertTrue(field.getBoolean(zkClient));
   }
+
+  @Test
+  public void testShouldSupportReadingAndWritingLargeJobModels() {
```

Comment:
Why is this test in TestZkUtils? This should be for whichever class the job model is being read/written from (e.g. ZkJC)

Reply:
This test is not necessary anymore. This is covered by `TestZkMetadataStore`.